src/db/mongo/client.pl

```diff
@@ -255,14 +256,26 @@
 %
 % Restore mongo DB.
 %
-mng_restore(_DB,Dir) :-
+mng_restore(DB,Dir) :-
+  mng_uri(URI),
   process_create(path(mongorestore),
-    [ Dir ],
+    [ '--uri', URI, '--db', DB, '--dir', Dir ],
     [ process(PID) ]
   ),
   wait(PID,exited(0)).
 
 
+mng_uri(URI) :-
```

Comment:
Could you add a comment, and also export the predicate in the module definition at the top?